server/controllers/finance/account.js

```diff
@@ -0,0 +1,80 @@
+var db = require('../../lib/db');
+
+function create (req, res, next){
+  var record = req.body;
+  var create_account_query = 'INSERT INTO account SET ?';
+  var transaction = db.transaction();
```

Comment:
You are only executing one query - do you need a transaction?  A transaction will temporarily "lock" a table, which may prevent others from access it simultaneously.  Could you just use a regular `db.exec()` instead?